apps/src/aichat/views/AichatView.tsx

```diff
@@ -163,6 +164,8 @@ const AichatView: React.FunctionComponent = () => {
 
   const resetProject = useCallback(() => {
     dispatch(resetToDefaultAiCustomizations(levelAichatSettings));
+    // Save the customizations to the user's project.
+    dispatch(updateAiCustomization());
```

Comment:
Thanks for adding this! Reminded me of your comment about the start over string - https://github.com/code-dot-org/code-dot-org/pull/58944#discussion_r1625151333. Is this a good time to move string?

Reply:
Oh good call! Will do

Reply:
Thank you!